packages/client/hmi-client/src/workflow/ops/funman/tera-funman.vue

```diff
@@ -1,133 +1,143 @@
-							v-for="(cfg, index) in node.state.constraintGroups"
-							:key="index"
-							:config="cfg"
-							:index="index"
-							:model-node-options="modelNodeOptions"
-							@delete-self="deleteConstraintGroupForm"
-							@update-self="updateConstraintGroupForm"
-						/>
-
-						<Button label="Add another constraint" size="small" @click="addConstraintForm" />
 					</div>
-					<div class="container output">
-						<h4 class="primary-text">Validation results</h4>
-						<tera-funman-output v-if="outputId" :fun-model-id="outputId" />
-						<div v-else>
-							<img src="@assets/svg/plants.svg" alt="" draggable="false" />
-							<h4>No Output</h4>
+					<InputText
+						:disabled="true"
+						class="p-inputtext-sm timespan-list"
+						inputId="integeronly"
+						v-model="requestStepListString"
+					/>
+					<p v-if="!showAdditionalOptions" @click="toggleAdditonalOptions" class="green-text">
+						Show additional options
+					</p>
+					<p v-if="showAdditionalOptions" @click="toggleAdditonalOptions" class="green-text">
+						Hide additional options
+					</p>
+					<div v-if="showAdditionalOptions">
+						<div class="button-column">
+							<label>Tolerance</label>
+							<InputNumber
+								mode="decimal"
+								:min="0"
+								:max="1"
+								:min-fraction-digits="0"
+								:max-fraction-digits="7"
+								v-model="tolerance"
+							/>
 						</div>
-					</div>
-				</div>
-				<div v-else class="content">
-					<div class="container">
-						<!-- TODO: notebook functionality -->
-						<p>{{ requestConstraints }}</p>
-					</div>
-					<div class="container">
-						<tera-funman-output v-if="outputId" :fun-model-id="outputId" />
-						<div v-else>
-							<img src="@assets/svg/plants.svg" alt="" draggable="false" />
-							<h4>No Output</h4>
+						<Slider v-model="tolerance" :min="0" :max="1" :step="0.01" />
+						<div class="section-row">
+							<!-- This will definitely require a proper tool tip. -->
+							<label>Select parameters to synthesize <i class="pi pi-info-circle" /></label>
+							<div
+								v-for="(parameter, index) of requestParameters"
+								:key="index"
+								class="button-column"
+							>
+								<label>{{ parameter.name }}</label>
+								<Dropdown v-model="parameter.label" :options="labelOptions"> </Dropdown>
+							</div>
 						</div>
 					</div>
-				</div>
-				<div class="footer">
+					<div class="spacer">
+						<h4>Add sanity checks</h4>
+						<p>Model configurations will be tested against these constraints</p>
+					</div>
+					<tera-constraint-group-form
+						v-for="(cfg, index) in node.state.constraintGroups"
+						:key="index"
+						:config="cfg"
+						:index="index"
+						:model-node-options="modelNodeOptions"
+						@delete-self="deleteConstraintGroupForm"
+						@update-self="updateConstraintGroupForm"
+					/>
+
+					<Button label="Add another constraint" size="small" @click="addConstraintForm" />
+				</main>
+				<!-- <footer>
```

Comment:
Is there a reason to keep this comment footer here?

Reply:
No just forgot to remove them 👎